Final edits

zachary-foster opened the pull request:
I looked over everything on this repository, but not the example repositories. I rephrased and reformatted a few things; change em back if you disagree.

zachary-foster committed:
rephrased, added, and grammar changes mostly

zachary-foster committed:
starting edits

zachary-foster committed:
finished formatting edits

zachary-foster committed:
finished edits

zkamvar commented on 02--Setup.Rmd:
```diff
@@ -11,27 +11,29 @@
 ----
 
-The following packages are utilized in this primer:
+The following packages are used in this primer:
```
Good catch!

In almost all cases used >>> utilized.

zkamvar commented on 02--Setup.Rmd:
```diff
@@ -11,27 +11,29 @@
-science in R. Before we get started, we should install the necessary components
-for this tutorial.
+This website is an introduction to some of the principles of reproducible
+science in R, not R itself or associated software and extensions (i.e. "packages").
```
This sounds like legalese. 😧

zachary-foster replied:
Yea, your version is better

zkamvar commented on tutorial.css:
```diff
@@ -80,9 +80,9 @@ in the center, so it's placed to the left. These are used with .knit_border and
 #exercise {
   display: block;
-  width: 98%;
+  width: 100%;
```
I placed this at 98% because the iframe was cutting off the border at 100%

zkamvar replied:
Just tried building the website with quilt(). 

Apparently this works. I wonder why it didn't before

zachary-foster replied:
Hmm, not sure why that would be. I have notice odd stuff occasionally, but there should be JavaScript to scale the iframes automatically.

zkamvar committed:
remove legalese

zkamvar committed:
Add div class for r-help-page

I'm experimenting with the package printr to print
a help page on the tutorial. These div tags help
separate the r-help-page from the rest of the
document.

I lifted it from the css at the top of
http://yihui.name/printr/

zkamvar committed:
Clarify setup chapter

 - utilize printr for displaying a help page
 - change example from getwd() to sessionInfo()
   IMO, this is MUCH more useful than getwd() in
   the context of reproducible research
 - change help topic from poppr to tidyr
   Not everyone is a population geneticist, but
   everyone could use tidy data!
 - display the help page for tidyr's gather
 - add an example of how to see examples :)

zkamvar commented:
I've clarified the setup chapter a bit more. To do this, I'm using the printr package: https://github.com/yihui/printr, which I think is great!
- utilize printr for displaying a help page
- change example from getwd() to sessionInfo()
  IMO, this is MUCH more useful than getwd() in
  the context of reproducible research
- change help topic from poppr to tidyr
  Not everyone is a population geneticist, but
  everyone could use tidy data!
- display the help page for tidyr's gather
- add an example of how to see examples :)

What do you think? If you like it, merge it.

zachary-foster commented:
I think all that looks good. The `printr` package looks cool. Good stuff.

zkamvar commented:
Would you like me to push the gh-pages branch?

zachary-foster commented:
Yea, I still cant get the image css to apply correctly. Thanks
